extensions/wikia/Email/SocialLinksGenerator.class.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class SocialLinksGenerator
+ * @package Email
+ */
+class SocialLinksGenerator {
+	const SOCIAL_MEDIA = [
+		'en' => [
+			'Facebook' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Facebook" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)